vizro-core/docs/pages/user_guides/tabs.md

```diff
@@ -0,0 +1,121 @@
+                                        title="Graph 2",
+                                        x="continent",
+                                        y="lifeExp",
+                                        color="continent",
+                                    ),
+                                ),
+                            ],
+                        ),
+                        vm.Container(
+                            title="Tab II",
+                            components=[
+                                vm.Graph(
+                                    figure=px.scatter(
+                                        gapminder_2007,
+                                        title="Graph 3",
+                                        x="gdpPercap",
+                                        y="lifeExp",
+                                        size="pop",
+                                        color="continent",
+                                    ),
+                                ),
+                            ],
+                        ),
+                    ],
+                ),
+            ],
+            controls=[vm.Filter(column="continent")],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+
+        Vizro().build(dashboard).run()
+        ```
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration
+        # See from_yaml example
+        pages:
+          - components:
+              - tabs:
+                  - title: Tab I
+                    type: container
+                    components:
+                      - type: graph
+                        figure:
+                            _target_: bar
+                            data_frame: gapminder_2007
+                            title: Graph 1
+                            x: continent
+                            y: lifeExp
+                            color: continent
+                      - type: graph
+                        figure:
+                            _target_: box
+                            data_frame: gapminder_2007
+                            title: Graph 2
+                            x: continent
+                            y: lifeExp
+                            color: continent
+                  - title: Tab II
+                    type: container
+                    components:
+                      - type: graph
+                        figure:
+                            _target_: scatter
+                            data_frame: gapminder_2007
+                            title: Graph 3
+                            x: gdpPercap
+                            y: lifeExp
+                            size: pop
+                            color: continent
+              type: tabs
+          controls:
+              - type: filter
+                column: continent
+          title: Tabs
+        ```
+    === "Result"
```

Comment:
I realize we need gifs... a lot of the advanced functionality just doesn't come across anymore.. discussed this also with Adam. But for a separate PR